weaviate/collection/grpc_base.py

```diff
@@ -0,0 +1,570 @@
+from dataclasses import dataclass
```

Comment:
let's name this different, we have a class `_BaseGRPC` that is in `.grpc_shared` and in the file `grpc_base.py` is the class `_GRPC`

Reply:
Agreed, will do now!

Reply:
There's more renaming that should be done as mentioned in https://github.com/weaviate/weaviate-python-client/pull/531 but I'll leave it for now